introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi, my name is Nicolò Scarpa and i'm Happy to be introduced to the dojo.
 My favorite Code School path is JavaScript.
+My deadly skills are: 
+* PHP
+* MVC design
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?